demos/python_demos/machine_translation_demo/machine_translation_demo.py

```diff
@@ -0,0 +1,227 @@
+        self.logger.info("path: " + path)
+        self.logger.info("max_tokens: " + str(max_tokens))
+        self.tokenizer = SentencePieceBPETokenizer(
+            os.path.join(path, "vocab.json"),
+            os.path.join(path, "merges.txt")
+        )
+        self.max_tokens = max_tokens
+        self.idx = {}
+        for s in ['</s>', '<s>', '<pad>']:
+            self.idx[s] = self.tokenizer.token_to_id(s)
+
+    def encode(self, sentence):
+        """ Encode method for sentence.
+
+        Arguments:
+            sentence (str): sentence.
+
+        Returns:
+            tokens (np.array): encoded sentence in tokneized format.
+        """
+        tokens = self.tokenizer.encode(sentence).ids
+        tokens = self._extend_tokens(tokens)
+        return tokens
+
+    def decode(self, tokens, remove_repeats=True):
+        """ Decode method for tokens.
+
+        Arguments:
+            tokens (np.array): sentence in tokenized format.
+            remove_repeats (bool): remove repeated words.
+
+        Returns:
+            sentence (str): output sentence.
+        """
+        sentence = self.tokenizer.decode(tokens)
+        for s in self.idx.keys():
+            sentence = sentence.replace(s, '')
+        if remove_repeats:
+            sentence = self._remove_repeats(sentence)
+        return sentence.lstrip()
+
+    def _extend_tokens(self, tokens):
+        """ Extend tokens.
+
+        Arguments:
+            tokens (np.array): sentence in tokenized format.
+
+        Returns:
+            tokens (np.array): extended tokens.
+        """
+        tokens = [self.idx['<s>']] + tokens + [self.idx['</s>']]
+        pad_length = self.max_tokens - len(tokens)
+        if pad_length > 0:
+            tokens = tokens + [self.idx['<pad>']] * pad_length
+        return tokens
+
+    def _remove_repeats(self, sentence):
+        """ Remove repeated words.
+
+        Arguments:
+            sentence (str): sentence.
+
+        Returns:
+            sentence (str): sentence in lowercase without repeated words.
+        """
+        tokens = sentence.lower().split()
+        return " ".join(key for key, _ in itertools.groupby(tokens))
+
+
+def build_argparser():
+    """ Build argument parser.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-m", "--model", required=True, type=str,
+                        help="Required. Path to an .xml file with a trained model")
+    parser.add_argument('--tokenizer-src', type=str, required=True,
+                        help='Required. Path to the folder with src tokenizer that contains vocab.json and merges.txt.')
+    parser.add_argument('--tokenizer-tgt', type=str, required=True,
+                        help='Required. Path to the folder with tgt tokenizer that contains vocab.json and merges.txt.')
+    parser.add_argument('--output-name', type=str, default='pred/Squeeze',
+                        help='Name of the models output node.')
```

Comment:
```suggestion
                        help='Optional. Name of the models output node.')
```
Help list in readme should be updated after that

Reply:
done